### PR TITLE
Add Web Bluetooth API in new Hardware category

### PR DESCRIPTION
**Pull request by beaufortfrancois**
I'd love to know what is Mozilla's opinion on the Web Bluetooth API project.

https://www.w3.org/community/web-bluetooth/
https://github.com/WebBluetoothCG/web-bluetooth

### CODE REVIEW

File: features.json [hunk 1]
```diff
@@ -346,6 +346,19 @@
...
+  "Hardware": [
+    {
+      "name": "Web Bluetooth",
+      "contact": "",
+      "status": "in progress|favorable|not favorable|no opinion",
+      "description": "",
+      "urls": {
+        "bugzilla": "",
+        "documentation": "",
+        "specification": ""
```
Comment (marcoscaceres) [inline features.json]: https://webbluetoothcg.github.io/web-bluetooth/

File: features.json [hunk 2]
```diff
@@ -346,6 +346,19 @@
...
+      }
+    }
+  ]
```
Comment (marcoscaceres) [inline features.json]: missing ","

**Comment by beaufortfrancois** [comment]
Thanks @marcoscaceres, I've addressed both issues.

**Comment by rik** [comment]
@marcoscaceres Do you know who could answer this?

**Comment by beaufortfrancois** [comment]
FYI https://developers.google.com/web/updates/2015/07/interact-with-ble-devices-on-the-web

**Comment by jyasskin** [comment]
@shuangMoz maybe.

**Comment by shuangMetiz** [comment]
Our current plan is to align with Web Bluetooth GATT client API and work on GATT server part with jyasskin. @yrliou is working on both GATT client/server API for FirefoxOS.

**Comment by marcoscaceres** [comment]
@shuangMoz thanks :)

**Comment by beaufortfrancois** [comment]
@marcoscaceres Shall I edit the commit and include `"favorable"` then?

**Comment by beaufortfrancois** [comment]
(gentle ping)

**Comment by beaufortfrancois** [comment]
@Rik Can you have a look at this?

**Comment by beaufortfrancois** [comment]
@Rik @marcoscaceres (gentle ping)

**Comment by marcoscaceres** [comment]
@beaufortfrancois sorry, yes: "favorable" :+1:

**Comment by beaufortfrancois** [comment]
@marcoscaceres I've edited my commit. Please merge at your convenience.

For info, is the 'features.json" file consumed/used already?

**Comment by rik** [comment]
I'm not working on this project anymore. Other Mozilla employees are now conducting this project over at https://github.com/mozilla/platatus. I suggest submitting a pull request over there. Sorry for this.

**Pull request closed without merging**